### DISCUSSION TITLE
Force "revalidate" of static page

### DISCUSSION THREAD

**Discussion by simplenotezy** [vercel/next.js#25543] [Help]
When using `getStaticProps` how can I force a revalidation to happen? I know I can leverage preview mode, however, once the content editor is done previewing, and would like to force the new content to be created, how can I tell Next.js to revalidate the specific page?

I guess I could issue a new build, but it would be more ideal to have it revalidate just the specific page, without having to go through a redeployment.

**Comment by sanzhardanybayev** [comment]
@simplenotezy I guess most people would just love this feature. Cause having server to revalidate very often increases server load.  Especially when you have ton of data per page.

**Reply by simplenotezy** [reply]
I worked around it by setting revalidate to 1, and then have Cloudflare Cache API cache the generated page, and serve it upon future requests. If I wish to force a revalidate, I can clear the cache for the whole zone/domain, or simply purge a single page by its URL. It seems to work pretty neat.

I then use Cloudflare Workers to modify the response header, so the max-age is not 1, but something long in the future.
